boards/amd/versalnet_rpu/versalnet_rpu_defconfig

```diff
@@ -0,0 +1,18 @@
+CONFIG_ISR_STACK_SIZE=1024
+CONFIG_THREAD_STACK_INFO=y
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable serial port
+CONFIG_UART_PL011=y
+
+CONFIG_ARM_ARCH_TIMER=y
+
+CONFIG_ARM_MPU=y
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=100000000
```

Comment:
why can't this be set at soc level?

Reply:
CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=100000000 - this is configurable based on HW design and not really hardcoded in SOC.

Reply:
will read it from dt parameter